app/views/my/page_layout.html.erb

```diff
@@ -27,6 +27,7 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 <script language="JavaScript">
```

Comment:
I believe you need to remove the previous script line.

Reply:
Good catch, thanks!